uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-balance-tab.component.spec.js

```diff
@@ -203,7 +202,11 @@ define([
 
       it('loads the balance report for contacts managed by the user, on selected absence period and type, on page 1, with a limited amount of records', function () {
         expect(leaveBalanceReport.all).toHaveBeenCalledWith(
-          expectedFilters,
+          {
+            period_id: absencePeriod.id,
+            type_id: absenceType.id,
+            managed_by: loggedInContactId
+          },
```

Comment:
why not use `expectedFilters`?

Reply:
expectedFilters and actual filters passed to API have different props. While I decided it is better to keep `absence_period` prop for component filters because it sounds more self-explanatory, API actually requires `period_id` to be passed. Same for `absence_type` and `type_id`.

Reply:
since you are not using `expectedFilters` in the expectation (only on the `$broadcast` call), change the name to `filters` or `filterValues`, etc.